public/static/docs/command-reference/get.md

```diff
@@ -173,12 +174,11 @@ $ dvc get . model.pkl --rev 7-train --out model.monograms.pkl
 
 The `model.monograms.pkl` file now contains the older version of the model. To
 get the most recent one, we use a similar command, but with
-
-`-o model.bigrams.pkl` and `--rev 9-bigrams-model` or even without `--rev`
-(since it's the latest version anyway). In fact, in this case using `dvc pull`
-with the corresponding [DVC-files](/doc/user-guide/dvc-file-format) should
-suffice, downloading the file as just `model.pkl`. We can then rename it to make
-its version explicit:
+`-o model.bigrams.pkl` and `--rev 9-bigrams-model` (or even without `--rev`
```

Comment:
we should use human-readable tag names in example for bigrams and baseline (not these with numbers)

Reply:
Hmmmm, but the previous paragraph reads

> If you are familiar with our Get Started example, , you may know that each chapter has a corresponding tag. Tag `7-train` is where we train a first version of the example model, and tag `9-bigrams-model` has an improved model (trained using bigrams).

And then those numbered tags are used in the following paragraphs including the one you commented on. I would need to change to `baseline-experiment`and `bigrams-experiment` everywhere and that first paragraph would no longer make much sense. I could rewrite it though. Lmk!

p.s. this is actually the only file where we use the numbered tags, all other doc examples use the more descriptive labels.

Reply:
On this one, I decided to revise that earlier paragraph in order to use `baseline-experiment` and `bigrams-experiment` throughout the doc. See 70f9ad5e